packages/shared/src/storage/AppleKeyChainStorage/AppleKeyChainStorage.ts

```
@@ -0,0 +1,65 @@
+  private getKeychainModule(): IAppleKeyChainNativeModule {
+    if (platformEnv.isNativeIOS) {
+      const m = NativeModules?.KeychainModule;
+      if (!m) {
+        throw new OneKeyLocalError('Keychain native module not found');
+      }
+      return m;
+    }
+    if (platformEnv.isDesktopMac) {
+      return desktopApiProxy.keychain;
+    }
+    throw new OneKeyLocalError('Failed to load Keychain module');
```

ESLint

[error] Expected an error object to be thrown.

(@typescript-eslint/no-throw-literal)